Sources/tart/Commands/Run.swift

```diff
@@ -71,6 +75,10 @@ struct Run: AsyncParsableCommand {
     } else if !noGraphics {
       runUI()
     }
+
+    if let vncWrapper = vncWrapper {
```

Comment:
Could you please rebase after #132 to make sure the VNC will be stopped after the VM is stopped and not right away.

Reply:
Done.